### DISCUSSION TITLE
source run_sim 環境問題

### DISCUSSION THREAD

**Discussion by michael61112** [bol-edu/HLS-SOC-Discussions#131] [Q&A]
想請問我在執行run_sim時，counter_la_fir.hex沒有產生出來，
我看log在一開始就有發生error了，
counter_la_fir.elf section `.data' will not fit in region `dff'
這是我設定環境的問題嗎?

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/cd3cb9db-c3a8-46af-a97b-63d17c83284a)

**Comment by ruei7916** [comment]
感覺是你的已初始化全域變數占太多空間了

**Comment by michael61112** [comment]
發現不是環境設定的問題，修改fir.c的內容後，此問題以解
